ci/do_ci.sh

```diff
@@ -215,7 +215,7 @@ elif [[ "$1" == "bazel.coverage" ]]; then
 
   # gcovr is a pain to run with `bazel run`, so package it up into a
   # relocatable and hermetic-ish .par file.
-  bazel build @com_github_gcovr_gcovr//:gcovr.par
+  bazel build --incompatible_remove_old_python_version_api=false @com_github_gcovr_gcovr//:gcovr.par
```

Comment:
Is there a tracking bug to clean this one up? Hoping not to be too surprised at a Bazel point release.